docs/api/profiles.md

```diff
@@ -0,0 +1,86 @@
+# Profiles
+
+Profiles contain information about a User. A User does not need a Profile, but a Profile is always created
+together with a User.
+
+## Get the profile of a user
+
+### URL
+
+```http
+GET /profiles/:profile HTTP/1.1
```

Comment:
the `user/` link is only for the current user. Unless you mean `users/profile` which is a good suggestion.

You propose `users/:userId/profile` ?